Assets/Src/GameplayPresenter/UnitsUnderlines/UnitsUnderlinesPresenter.cs

```diff
@@ -0,0 +1,39 @@
+﻿using castledice_game_logic;
+using castledice_game_logic.GameObjects;
+using Src.GameplayView.UnitsUnderlines;
+
+namespace Src.GameplayPresenter.UnitsUnderlines
+{
+    public class UnitsUnderlinesPresenter
+    {
+        private readonly IUnitsUnderlinesView _view;
+        
+        public UnitsUnderlinesPresenter(Board board, IUnitsUnderlinesView view)
+        {
+            _view = view;
+            foreach (var cell in board)
+            {
+                cell.ContentAdded += OnContentAdded;
+                cell.ContentRemoved += OnContentRemoved;
```

Comment:
We should introduce unsibscribe machanism in a desctructor or with alternative way